tests/test_models/test_city.py

```diff
@@ -5,15 +5,15 @@
 
 from datetime import datetime
 import inspect
-import models
 from models import city
```

Comment:
This line is no longer useful and should be removed.

Reply:
Based on the previous work with shortening base_model.py imports, this can be shortened even more to:
from models import storage_t
from models.city import City, __doc__ as city_doc